packages/components/src/components/popover/popover.jsx

```diff
@@ -132,14 +132,29 @@ const Popover = ({
                               },
                           }
                         : { containerStyle: { zIndex } })}
-                    content={({ position, targetRect, popoverRect }) => {
+                    content={({ position, childRect, popoverRect }) => {
                         return (
                             <ArrowContainer
                                 position={position}
-                                targetRect={targetRect}
+                                childRect={childRect}
                                 popoverRect={popoverRect}
                                 arrowColor={has_error ? 'var(--status-danger)' : 'var(--general-active)'}
                                 arrowSize={5}
+                                arrowStyle={
```

Comment:
there was a bug in `popover` after I updated it and the arrow icon was missing( it was placed out of the screen) and I couldn't fix it So I had to add a custom style for it